src/deployconfig.jl

```diff
@@ -843,6 +843,200 @@ authentication_method(::Buildkite) = Documenter.SSH
 
 documenter_key(::Buildkite) = ENV["DOCUMENTER_KEY"]
 
+#################
+# Woodpecker CI #
+#################
+
+"""
+    Woodpecker <: DeployConfig
+
+Implementation of `DeployConfig` for deploying from Woodpecker CI.
+
+The following environmental variables are built-in from the Woodpecker pipeline
+influences how `Documenter` works:
+ - `CI_REPO`: must match the full name of the repository <owner>/<name> e.g. `JuliaDocs/Documenter.jl`
+ - `CI_BUILD_EVENT`: must be set to `push`, `tag`, `pull_request`, and `deployment`
+ - `CI_COMMIT_REF`: must match the `devbranch` keyword to [`deploydocs`](@ref), alternatively correspond to a git tag.
+
+User can define the `FORGE_URL` variable and add it to their Woodpecker pipeline definition:
+
+Example `.woodpecker.yml`
+```yaml
+pipeline:
+   docs:
+   image: julia
+   environment:
+     - FORGE_URL=github.com
+   ...
+```
+
+Or
+
+```yaml
+pipeline:
+   docs:
+   image: julia
+   commands:
+     - export FORGE_URL=github.com
+   ...
+```
+
+More about pipeline syntax is documented here: https://woodpecker-ci.org/docs/usage/pipeline-syntax
+
+Lastly, another environment-variable used for authentication is
+the `PROJECT_ACCESS_TOKEN` which is an access token you defined by 
+the forge you use e.g. GitHub, GitLab, Codeberg, and other gitea 
+instances. Check their documentation on how to create an access token. 
+This access token should be then added as a secret as documented in 
+https://woodpecker-ci.org/docs/usage/secrets. 
```

Comment:
The markdown parser doesn't pick up links automatically.

```suggestion
More about pipeline syntax is documented here: <https://woodpecker-ci.org/docs/usage/pipeline-syntax>

Lastly, another environment-variable used for authentication is
the `PROJECT_ACCESS_TOKEN` which is an access token you defined by 
the forge you use e.g. GitHub, GitLab, Codeberg, and other gitea 
instances. Check their documentation on how to create an access token. 
This access token should be then added as a secret as documented in 
<https://woodpecker-ci.org/docs/usage/secrets>.
```